CMakeLists.txt

```diff
@@ -59,9 +59,9 @@ if (WIN32)
     include_directories(${DirectX_INCLUDE_DIR})
 
     set(CMAKE_EXE_LINKER_FLAGS_DEBUG "${CMAKE_EXE_LINKER_FLAGS_DEBUG}          /DEBUG /LARGEADDRESSAWARE ")
-    set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE}        /DEBUG /LARGEADDRESSAWARE /OPT:REF /LTCG:STATUS /RELEASE")

```

Comment:
We want to keep `/LTCG` - it's "whole program optimization". I haven't measured but theoretically it could make big difference. It's OK to remove the `:STATUS` part tho, that's just a build log setting.